Gemfile

```diff
@@ -1 +1 @@
-gem 'rubocop', '>= 1.0', '< 2.0'
\ No newline at end of file
+# gem 'rubocop', '>= 1.0', '< 2.0'
```

Comment:
[OPTIONAL] Since it's not held in the project you are free to comment it. However, it would be nice if you could remove any commented code or uncomment in this case would be much apprecated.